codespell_lib/data/dictionary.txt

```diff
@@ -26928,21 +27125,64 @@ piars->pairs, piers, pliers,
 piblisher->publisher
 pice->piece
 pich->pitch, pick, pinch,
+piched->pitched, picked, pinched,
+piches->pitches, pinches,
+piching->pitching, picking, pinching,
+picknic->picnic
+pickniced->picnicked
+picknicer->picnicker
+picknicing->picnicking
+picknicked->picnicked
+picknicker->picnicker
+picknicking->picnicking
+picknicks->picnics
+picknics->picnics
+pickyune->picayune
+pickyunes->picayunes
+picniced->picnicked
+picnicer->picnicker
+picnicing->picnicking
+picnick->picnic, picknick,
+picnicks->picnics
 picoseond->picosecond
 picoseonds->picoseconds
+picturesk->picturesque
+pictureskely->picturesquely
+pictureskly->picturesquely
+pictureskness->picturesqueness
 pieceweise->piecewise, piece wise,
 piecewiese->piecewise, piece wise,
 piecwise->piecewise, piece wise,
+pigen->pigeon
```

Comment:
Could be `pigpen` too. Same for plural below.

Reply:
Added